docs/development/examples/nearlib/introduction.md

```diff
@@ -235,7 +235,7 @@ If you're already familiar with these terms then feel free to skip ahead.
   Accounts must pay the network for their own storage and compute as well as fees for processing transactions they initiate
 
 - **the cost of data storage and computation on the network is measured in units of gas which is a configurable bit of network economics but is usually vanishingly small**  \
-  It costs exactly 2 gas (at the time of this writing) to "compute the creation of a new account" on the network where 1 unit of gas is equivalent to 1 attoNEAR.  For a sense of scale, with the 10 NEAR in your new account you could happily afford to create a *new account for every grain of sand on the planet* (estimates suggest about 10^18 grains of sand) and still have some NEAR tokens left over for a rainy day because 1 NEAR is 10^18 attoNEAR
+  It costs exactly 2 gas (at the time of this writing) to "compute the creation of a new account" on the network where 1 unit of gas is equivalent to 1 yoctoNEAR.  For a sense of scale, with the 10 NEAR in your new account you could happily afford to create a *new account for every grain of sand on the planet* (estimates suggest about 10^24 grains of sand) and still have some NEAR tokens left over for a rainy day because 1 NEAR is 10^24 yoctoNEAR
```

Comment:
the popular estimate for grains of sand on earth is still 10^18 - 10^21 so changing it to 10^24 seems too far afield.

here are a few suggestions for rewriting this statement whose intention is to make these impossible scales relatable.  which do you like best?  i think picking one is enough.

_quantity_

> For a sense of scale, with just 2 of the 10 NEAR in your new account you could happily afford to create a *new account for every grain of sand on the planet* (actually estimates suggest about 10^21 grains of sand on earth so you could do this every day for 3 years, not just once) and still have some NEAR tokens left over for a rainy day.  Gas is measured in yoctoNEAR or yNEAR which is 10^24 times smaller than NEAR.  It's very very very small.

_mass_

> For a sense of scale, if NEAR were a unit of mass and we set 1 NEAR to be the mass of your laptop then 1 unit of gas would be about the mass of a proton.  Gas is measured in yoctoNEAR or yNEAR which is 10^24 times smaller than NEAR.  It's very very very small.

> For a sense of scale, if NEAR were a unit of mass and we set 1 NEAR to the mass of this planet earth we're all living on then 1 unit of gas would be about the mass of your best friend.  Gas is measured in yoctoNEAR or yNEAR which is 10^24 times smaller than NEAR.  It's very very very small.

_time_

> For a sense of scale, if we invited 1000 of our closest friends to a party where, according to the rules of a very cruel game, every one of us has to spend 1 _billion_ gas every second then it would take us about the same as all of recorded human history to get through 1 NEAR.  Gas is measured in yoctoNEAR or yNEAR which is 10^24 times smaller than NEAR.  It's very very very small.